### PR TITLE
Refactors the conditional again

### PR DESCRIPTION
**Pull request by itsjoekent**
#### What's this PR do?

tell us, what have you changed?
#### How should this be reviewed?

tell us, how can we review, to test that this works
#### Any background context you want to provide?

what else?
#### Relevant tickets

Fixes #
#### Checklist
- [ ] Documentation added for new features/changed endpoints.
- [ ] Tested on staging.
- [ ] Pinged a PM if this is a larger PR that would benefit from some additional testing love.

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/js/components/InstructionSlide.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ class InstructionSlide extends React.Component {
             <div className="container__row">
               <div className="container__block -narrow -centered">
                 <h2 className="heading -gamma">{title}</h2>
-                {competition.joined ? <p>{competition.messages.confirmation}</p> : ''}
+                {competition && competition.joined ? <p>{competition.messages.confirmation}</p> : ''}
```
Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/js/components/InstructionSlide.js:26]: any chance we'd get `undefined` errors from `competition.messages.confirmation` not being present?

Comment (itsjoekent) [inline lib/themes/dosomething/paraneue_dosomething/js/components/InstructionSlide.js:26]: confirm msg is required 

from the signup data form thingy

```
  if ($values[$prefix . 'status'] == 1) {
    // Confirm_msg is mandatory.
    $confirm_msg = $prefix . 'confirm_msg';
    if (empty($values[$confirm_msg])) {
      form_set_error($confirm_msg, t('Signup Data Form Confirmation message is mandatory.'));
    }
  }
```

Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/js/components/InstructionSlide.js:26]: k cool

**Comment by weerd** [comment]
👍

**Pull request merged**